src/request/remove-key/index.html

```diff
@@ -1,15 +1,16 @@
 <!DOCTYPE html>
 <html>
+
```

Comment:
Why add an empty new line?

Reply:
(*I thought you liked them*)
My editor adds it when autoformating. I guess it's kind of a replacement for the missing indent, and adds some visual structure. But I don't care too much.